### PR TITLE
Add more Brazil react meetups

### PR DESCRIPTION
**Pull request by lucianomlima**

### CODE REVIEW

File: content/community/meetups.md [hunk 1]
```diff
@@ -22,8 +22,14 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
 ## Brazil {#brazil}
 * [Belo Horizonte](https://www.meetup.com/reactbh/)
 * [Curitiba](https://www.meetup.com/pt-br/ReactJS-CWB/)
+* [Florianópolis](https://www.meetup.com/pt-br/ReactJS-Floripa/)
+* [Goiânia](https://www.meetup.com/pt-br/ReactJS-Goiania/)
```
[the remark this answers is not in the document]
Comment (jhonmike) [reply content/community/meetups.md]: 😮 probably was renamed the group... can validate @lucianomlima?

[the remark this answers is not in the document]
Comment (lucianomlima) [reply content/community/meetups.md]: Yeah! Changed the name! I'll need to adjust on pt_BR version too 😬

[the remark this answers is not in the document]
Comment (lucianomlima) [reply content/community/meetups.md]: Fixed!

File: content/community/meetups.md [hunk 2]
```diff
@@ -22,8 +22,14 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
 ## Brazil {#brazil}
 * [Belo Horizonte](https://www.meetup.com/reactbh/)
 * [Curitiba](https://www.meetup.com/pt-br/ReactJS-CWB/)
+* [Florianópolis](https://www.meetup.com/pt-br/ReactJS-Floripa/)
+* [Goiânia](https://www.meetup.com/pt-br/React-Goiania/)
 * [Joinville](https://www.meetup.com/pt-BR/React-Joinville/)
+* [Juiz de Fora](https://www.meetup.com/pt-br/ReactJS-Juiz-de-Fora/)
```
Comment (lex111) [inline content/community/meetups.md]: O grupo que você está procurando não existe.

🙄

Comment (jhonmike) [reply content/community/meetups.md]: https://www.meetup.com/pt-BR/React-Juiz-de-Fora/

Comment (lucianomlima) [reply content/community/meetups.md]: All checked and fixed!

**Comment by lex111** [comment]
@lucianomlima thanks!

**Pull request merged**